dev-python/colorful/colorful-0.5.5.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 2019-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DISTUTILS_USE_PEP517=setuptools
+PYTHON_COMPAT=( python3_{10..12} pypy3 )
+
+inherit distutils-r1
+
+DESCRIPTION="Terminal string styling done right, in Python."
+HOMEPAGE="
+	https://pypi.org/project/colorful/
+	https://github.com/timofurrer/colorful
+"
+SRC_URI="
+	https://github.com/timofurrer/colorful/archive/refs/tags/v${PV}.tar.gz
+		-> ${P}.gh.tar.gz
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+RESTRICT="!test? ( test )"
```

Comment:
These two are added by `distutils_enable_tests` as well. Otherwise it couldn't add the dep.